Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM golang:1.6
 
-RUN go get -u github.com/segment-sources/postgres
 
-# Additionally add a cron-like runner to run on an
-# interval.
-RUN go get -u github.com/segmentio/go-every
```

Comment:
it's there in mongo source too. i'd keep it for this PR in case people are doing `docker run --entrypoint=every postgres --...` or something